spec/comma/sanitized_data_extractor_spec.rb

```diff
@@ -103,11 +103,12 @@
         name 'name' do |name| '+somestring' end
         name 'name' do |name| '-@1morestr1n6' end
         name 'name' do |name| '+1234567890' end
+        name 'name' do |name| '-.50' end
       end
     end.new(1).to_comma_sanitized
   end
 
-  it 'removes special characters for non digits and leaves only digits alone' do
-    @data.should eq(["somestring", "1morestr1n6", "+1234567890"])
+  it 'prepends an apostrophe in front of values that start with a special character unless it is only digits' do
+    @data.should eq(["'+somestring", "'-@1morestr1n6", "+1234567890", "'-.50"])
```

Comment:
I would have expected `"'+1234567890"` instead of `"+1234567890"` i.e. prepended with apostrophe in this case also. Does this pass?

Reply:
This PR left in place the phone number checking https://github.com/chargify/comma/blob/1f04b20f055971e0947c455ffc90649b0b0df1e0/lib/comma/sanitized_extractor.rb#L31  

That being said, my test description didnt articulate that at all, so seems unreasonable to expect you to assume it.